sdgym/synthesizers/base.py

```diff
@@ -146,6 +146,8 @@ class MultiSingleTableBaselineSynthesizer(BaselineSynthesizer, abc.ABC):
 
     These classes model and sample each table independently and then just
     randomly choose ids from the parent tables to form the relationships.
+
+    NOTE: doesn't currently work.
```

Comment:
Out of curiosity, did it work before ? If so do we know why is not currently working ?